doc/md/ci.mdx

```diff
@@ -94,3 +92,167 @@ fi
 
 </TabItem>
 </Tabs>
+
+## Lint migration files
+
+Changes to your project's Ent schema almost always result in a modification
+of your database. If you are using [Versioned Migrations](/docs/versioned-migrations)
+to manage changes to your database schema, we recommend running [migration linting](https://atlasgo.io/versioned/lint)
+as part of your continuous integration flow. This is done for multiple reasons:
+
+* Linting replays your migration directory on a real database to make sure all SQL statements
```

Comment:
what do you mean by real? let's call it a "container" and link to the Atlas dev-database doc.